e2e-tests/quote/cypress/e2e/journeys/quote/your-quote/your-quote-change-answers-single-policy.spec.js

```diff
@@ -179,7 +179,7 @@ context('Your quote page - change answers (single policy type to multiple policy
 
       const insuredFor = summaryList.field(QUOTE.INSURED_FOR);
 
-      cy.checkText(insuredFor.value(), '£84,000.00');
+      cy.checkText(insuredFor.value(), '£108,000.00');
```

Comment:
missed this, can this be a constant?

Reply:
Would be great - we have about 30 instances of such assertions, for prices that are ultimately generated from the pricing grid. Constants might actually hinder these assertions when they're updated, e.g what could we call these as constants? 🤔 

```
'£11,111,110.20'
'£143,209.86'
```

I think this should be separate from this PR as there are many instances and this PR unblocks a ticket that's been in sprints for ~4 months